src/main/java/javaserver/RequestHandlers/RequestHandler.java

```diff
@@ -0,0 +1,39 @@
+package javaserver.RequestHandlers;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+import javaserver.Request;
+
+public class RequestHandler {
```

Comment:
It also seems strange that this appears to be the base request handler, and yet its setup to handle the route for `/form`. The name seems a bit misleading, especially since all handlers appear to extend this handler. I would rethink this design and consider using an interface instead.